tools/README.md

```diff
@@ -83,3 +83,51 @@ python3 /graphstorm/tools/partition_graph.py --dataset ogbn-arxiv \
                                              --output /tmp/ogbn_arxiv_nc_train_val_1p_4t  \
                                              --is-homo     
 ```
+
+## Convert node features from distDGL format WholeGraph format
+
+Use the `convert_feat_to_wholegraph.py` script with `--dataset-path` pointing to the distDGL folder of partitions and `--feat-names` to the features you want to convert (e.g. '`paper/feat,author/feat,institution/feat`')
+```
+python3 convert_feat_to_wholegraph.py --dataset-path ogbn-mag240m-2p --feat-names paper/feat
```

Comment:
let's not expose internal node data names. you should use the same convention as training script. I believe the conversion is something like "paper:feat"

Reply:
Not exposing any names with the new design where it iterates through all the node feature names by default.